src/Symfony/Component/Notifier/Bridge/Slack/SlackTransport.php

```diff
@@ -72,19 +69,22 @@ protected function doSend(MessageInterface $message): void
         }
 
         $options = $opts ? $opts->toArray() : [];
-        $id = $message->getRecipientId() ?: $this->id;
+        $options['token'] = $this->accessToken;
+        if (!isset($options['channel'])) {
+            $options['channel'] = $message->getRecipientId() ?: $this->chatChannel;
+        }
         $options['text'] = $message->getSubject();
-        $response = $this->client->request('POST', sprintf('https://%s/services/%s', $this->getEndpoint(), $id), [
-            'json' => array_filter($options),
+        $response = $this->client->request('POST', 'https://'.$this->getEndpoint().'/api/chat.postMessage', [
+            'body' => array_filter($options),
```

Comment:
Why not as json?

Reply:
Great question. All this diff is from the 5.0 implementation.